### ISSUE TITLE
Request: ESP32-S3-DevKitC-1 board support

### ISSUE THREAD

**Issue by maumad** [platformio/platform-espressif32#674]
- [ ] **Feature Request**.
      It would be nice to have support to ESP32-S3-DevKitC-1 - MCU ESP32S3.

Best regards

**Comment by AndyW999** [comment]
Is this board request likely to be fulfilled for ESP_IDF?

Thanks

Andy.

**Comment by darkfiberiru** [comment]
@AndyW999 are you asking on platformio side or esp_idf side. 4.4 and 5.0 support esp32-s3. 4.4 is in beta and 5.0 is dev branch

**Comment by AndyW999** [comment]
ESP-IDF preferably.

**Comment by jonathandreyer** [comment]
Hi everyone,

I have also a ESP32-S3-DevKitC-1 and I have started the porting of this chip to PlatformIO (with ESP-IDF v4.4-beta). That was possible by the improvement made by @X-Ryl669 which enable the support of upcoming ESP-IDF version. Actually, it is a preliminary support and it doesn’t support ULP.

You can find the blink example [here](https://github.com/jonathandreyer/platformio-esp32s3-blink).

Best regards,

Jonathan Dreyer

**Comment by ralphrmartin** [comment]
It would also be nice to have support for the muselab nanoESP32-s3 board
[https://www.tindie.com/products/johnnywu/nanoesp32-s3-development-board/]
which I guess would only be a little extra effort once esp32-s3 tooling is in place.

**Comment by jonathandreyer** [comment]
Hi everyone,

I have updated the platform repository with ESP-IDF v4.4.1. I have tested it with the board mentioned in the title of the issue. To use it, it is only necessary to replace the platform by:
```
platform = https://github.com/jonathandreyer/platform-espressif32.git#espidf-441
```

Best regards,

Jonathan Dreyer

**Comment by jonathandreyer** [comment]
Hi everyone,

I did a quick update to inform you that the debug (by the internal JTAG interface) is also usable. The `upload_protocol` and `debug_tool` are `esp_usb_jtag`.

Best regards,

Jonathan Dreyer

**Comment by code-is-art** [comment]
This json from espressif has been updated to include the esp32-s3 dev board for arduino-esp32. I've been able to compile using the arduino ide for my esp32-s3 dev board. Not sure how to make it work for pio.
```
https://raw.githubusercontent.com/espressif/arduino-esp32/gh-pages/package_esp32_index.json
```

**Comment by kevinhikaruevans** [comment]
It looks like only Arduino is supported. Will there be support for the espidf?

**Comment by Jason2866** [comment]
@kevinhikaruevans #710 is still open. Until it is official released you can use my fork which supports Arduino core 2.0.3 and IDF 4.4.1. Using Arduino as Component of IDF does work too.
```
https://github.com/Jason2866/platform-espressif32
```
